CarpetXRegrid/configuration.ccl

```diff
@@ -0,0 +1,6 @@
+# Configuration definitions for thorn Regrid
+REQUIRES AMReX Arith Loop 
+
+PROVIDES Regrid
```

Comment:
The capability should probably be called `CarpetXRegrid` instead. It's not a standardized capability in the Einstein Toolkit.

Reply:
@stevenrbrandt